### PR TITLE
fix: "Not activated" UI is not shown WPB-7080

### PR DESCRIPTION
**Pull request by KaterinaWire**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-7080" title="WPB-7080" target="_blank">BugWPB-7080</a>  Client logged in with out E2Ei with an MLS thumbprint but still E2Ei - not activated UI is not shown on devices for other user
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

1. "Not activated" UI is not shown.
2. "Not activated" certificate should not have "Show certificate details" button.

### Solutions

Refactoring and bug fix in ` ClientListViewController.swift`

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by theswiftway** [comment]
please merge latest changes

File: wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/Components/DeviceDetailsButtonsView.swift [hunk 1]
```diff
@@ -83,8 +83,6 @@ struct DeviceDetailsButtonsView: View {
                     Divider()
                     getCertificateButton.padding()
                 }
-                Divider()
-                showCertificateButton.padding()
```
Comment (theswiftway) [inline wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/Components/DeviceDetailsButtonsView.swift:87]: https://www.figma.com/file/gSmh7BRyq9SGKkyOLSbOdM/E2E-Identity?type=design&node-id=234-34282&mode=design&t=I2XpMNbwg4u0aFxZ-0

Based on this UI we have to show Show certificate button

Comment (KaterinaWire) [reply wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/Components/DeviceDetailsButtonsView.swift:87]: It's a wrong UI. You don't have a certificate if the status is `Not Activated`. What should we show? 
The correct design is for Web

**Review by caldrian — APPROVED** [review]

**Review by theswiftway — APPROVED** [review]
LGTM

**Pull request merged**